src/Components/Patient/PatientRegister.tsx

```diff
@@ -397,8 +402,14 @@ export const PatientRegister = (props: PatientRegisterProps) => {
       if (!status.aborted) {
         if (res?.ok && data) {
           setPatientName(data.name || "");
+          if (data.year_of_birth) {
+            setAgeInputType("age");
+          }
```

Comment:
@rash-27 The backend will fill `year_of_birth` too when `date_of_birth` is given. But the opposite doesn't.


```suggestion
          if (!data.date_of_birth) {
            setAgeInputType("age");
          }
```

Reply:
Ok got it ,will change it